test/common/include/alpaka/test/event/EventHostManualTrigger.hpp

```diff
@@ -693,7 +694,7 @@ namespace alpaka
                 std::cerr << "[Workaround] polling of device-located value in stream, as hipStreamWaitValue32 is not "
                              "available.\n";
 #    endif
-                while(hostMem < 0x01010101u)
+                while(hostMem < 0x01010101)
```

Comment:
Why the change of type here?

Reply:
I guess because `hostMem` is also a signed integer. The previous version probably triggers a warning.

Reply:
implicit comparison between `int` and `unsigned int`. `hostMem` is `int` therefore the hex code must be `int` too.